tests/targets/test_android.py

```diff
@@ -52,11 +46,52 @@ def patch_platform(platform):
     return mock.patch("buildozer.targets.android.platform", platform)
 
 
-class TestTargetAndroid:
+def init_target(temp_dir, options=None):
+    buildozer = init_buildozer(temp_dir, "android", options)
+    return TargetAndroid(buildozer)
 
-    @staticmethod
-    def default_specfile_path():
-        return os.path.join(os.path.dirname(buildozer_module.__file__), "default.spec")
+
+def call_build_package(target_android):
+    """
+    Call the build_package() method of the tested TargetAndroid instance,
+    patching the functions that would otherwise produce side-effects.
+
+    Return the mocked execute_build_package() method of the TargetAndroid
+    instance so that tests can easily check which command-line arguments
+    would be passed on to python-for-android's toolchain.
+    """
+    buildozer = target_android.buildozer
+    expected_dist_dir = (
+        '{buildozer_dir}/android/platform/build-armeabi-v7a/dists/myapp__armeabi-v7a'.format(
+        buildozer_dir=buildozer.buildozer_dir)
+    )
+
+    with \
```

Comment:
Breaking the first line here seems a bit pointless, it actually leads to more indentation rather than less!

Reply:
I think it's because indentation needs to be a multiple a 4 and not under/over indented (unless the flake8 rule is commented which is the case) so if you want them aligned that's how you can address it.
I've changed it anyway, let me know what do you think